clusters/build-clusters/01_cluster/README.md

```diff
@@ -0,0 +1,36 @@
+# 01-Cluster
+
+[01-Cluster](URL_TBD) is an OpenShift-cluster managed by DPTP-team. It is one of the clusters for running Prow job pods.
+
+## Installation
+
+The aws account below is managed by DPP team with the public hosted zone (base domain for installer): TODO.
```

Comment:
Shall we wait for these TODOs in this PR?

Reply:
Yes